substrate/rpc/src/chain/mod.rs

```diff
@@ -51,11 +50,11 @@ build_rpc_trait! {
 
 		#[pubsub(name = "chain_newHead")] {
 			/// New head subscription
-			#[rpc(name = "subscribe_newHead")]
+			#[rpc(name = "subscribe_newHead", alias = ["chain_subscribeNewHead", ])]
```

Comment:
Great. Maybe it is better to have the "old" as the alias and the "new" one as the name?

(e.g. `chain_subscribeNewHead` is actually probably the preferred one to match since it aligns with other RPCs, my gut tells me the preferred one should be the "default")